kntest/pkg/cluster/ops/delete.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The Knative Authors
+Copyright 2020 The Knative Authors
```

Comment:
Don't change the copyright date; it should always be the earliest

Reply:
Changed back to 2019